docs/AboutUs.md

```diff
@@ -7,43 +7,48 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-### Isaac Lee
+### Lee Xiong Jie, Isaac
 
 <img src="images/luminousleek.png" width="200px">
 
-[[GitHub](https://github.com/luminousleek)]
+Role: Team Lead
 
-* Role: Team Lead
+* [GitHub](https://github.com/luminousleek)
+* [Project Portfolio Page](team/luminousleek.html)
 
-### Shezad Hassan
+### Shezad Sulyman Imran Hassan
 
 <img src="images/shezadhassan22.png" width="200px">
 
-[[GitHub](https://github.com/shezadhassan22)]
+Role: Testing
 
-* Role: Testing
+* [GitHub](https://github.com/shezadhassan22)
+* [Project Portfolio Page](team/shezadhassan22.html)
 
 ### Rohit Rajesh Bhat
 
 <img src="images/rohit0718.png" width="200px">
 
-[[GitHub](https://github.com/rohit0718)]
+Role: Documentation
 
-* Role: Documentation
+* [GitHub](https://github.com/rohit0718)
+* [Project Portfolio Page](team/rohit0718.html)
 
-### Benjamin Tan
+### TAN JUN RONG, BENJAMIN
```

Comment:
<img width="254" alt="image" src="https://user-images.githubusercontent.com/48402721/140656790-033319d2-a39e-47c9-bfb4-53b9ee115163.png">

Reply:
You are very rude. Please reflect on your behaviour!